### DISCUSSION TITLE
Add a json schema definition for workspace.json

### DISCUSSION THREAD

**Discussion by vymarkov** [nrwl/nx#5339] [Feature Requests]
Hi, at first I want to say a huge thank for this awesome project!

I wanna to clarify a question about json schema for workspace.json, I often use [JSON schema completion](https://nickymeuleman.netlify.app/blog/json-schema) to explore possible  options without need to leave my IDE but I didn't found a json schema definition for workspace.json, if this file is present somewhere, please let me know, thanks in advance!

**Comment by viceice** [comment]
yeah, a schema file for `nx.json` would help too, currently vscode maps it as `angular.json` file

![image](https://user-images.githubusercontent.com/1798109/161503689-bff4860b-9905-4e18-8651-fb301deeb0dc.png)


- https://github.com/nrwl/nx-console/issues/1263
